go/vt/tabletserver/mock/controller.go

```diff
@@ -0,0 +1,116 @@
+// Copyright 2015, Google Inc. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// Package mock provides mock interfaces for tabletserver.
+package mock
```

Comment:
This is way too generic of a name, I don't want to repeat the 'proto' subdir thing where we have to rename them on import everywhere... How about tabletservermock? Or tsmock is you want a shorter name...